pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -943,7 +955,7 @@ impl RemoteTimelineClient {
 
     /// Wait for all previously scheduled uploads/deletions to complete
     pub(crate) async fn wait_completion(self: &Arc<Self>) -> anyhow::Result<()> {
-        let receiver = {
+        let receiver: tokio::sync::watch::Receiver<()> = {
```

Comment:
What is this change for?

Reply:
yeah... rust-analyzer seems to be randomly adding these type defs now... https://github.com/neondatabase/neon/pull/7668/commits/eb8f9a39920dfff104fc8f61a6117e7d4fdcd6c3